pom.xml

```diff
@@ -1834,6 +1834,13 @@
                                     <!--<bundledSignature>jdk-reflection</bundledSignature>-->
 
                                     <!-- All following signatures should be replicated for each target JDK version we intend to support -->
+                                    <!--
+                                         Applying only signatures for the current JDK until this is fixed:
+                                         https://github.com/policeman-tools/forbidden-apis/issues/197
+                                         TODO : restore the previous config when forbidden-apis gets a fix.
+                                     -->
+                                    <bundledSignature>jdk-unsafe</bundledSignature>
```

Comment:
this is perfectly fine, there's no reason to change this. If you compile against Java 11 there cannot be any method from 17 used. Later jdk-unsafe won't remove methods.

Reply:
Right, thanks!